zebra/main.c

```diff
@@ -39,6 +39,7 @@
 #include "routemap.h"
 #include "frr_pthread.h"
 
+#include "zebra/zebra_router.h"
```

Comment:
Why router vs vrf? Let's say one day we support multiple logical routers/systems using a single zebra process, would the new structure still apply or just need to be changed/extended?

Reply:
The latter, right?

Reply:
@louberger -> with the changes to bring in namespace based vrf's the long term plans to support multiple logical routers in a single process has gone out the window from my perspective.  The current plan from my perspective is to run FRR inside of the individual processes to get the logical router.  I just don't see a good programming path forward with the current abstraction paradigm.  Perhaps this becomes clearer to me when I finish this work, as that there is still more work here to be done to finish massaging the code into the right structure, but I do not see it yet.